src/installer/tests/Assets/TestProjects/BundleProbeTester/Program.cs

```diff
@@ -13,16 +13,26 @@ public static class Program
         // The bundle-probe callback is only called from native code in the product
         // Therefore the type on this test is adjusted to circumvent the failure.
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        public delegate byte BundleProbeDelegate([MarshalAs(UnmanagedType.LPUTF8Str)] string path, IntPtr size, IntPtr offset);
+        public unsafe delegate byte BundleProbeDelegate([MarshalAs(UnmanagedType.LPUTF8Str)] string path, IntPtr offset, long* sizes);
```

Comment:
You use `IntPtr` for offset here, but the function pointer at the top is `uint64_t`. Won't that be the wrong size on x86?